server/netty-server/loom/src/main/scala/sttp/tapir/server/netty/loom/internal/ox/OxDispatcher.scala

```diff
@@ -0,0 +1,14 @@
+package sttp.tapir.server.netty.loom.internal.ox
+
+import ox.*
+import ox.channels.Actor
+
+private[loom] class OxDispatcher()(using ox: Ox) {
+  private class Runner {
+    def runAsync(thunk: Ox ?=> Unit): Unit =
+      fork(thunk(using ox)).discard
```

Comment:
weren't we supposed to create a new scope for running the `thunk`? also, what about error handling?